src/main/java/com/softdevsix/api/repositories/IProjectRepository.java

```diff
@@ -4,10 +4,13 @@
 import com.softdevsix.api.domain.entities.project.Project;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.UUID;
 
 @Repository
 public interface IProjectRepository {
     Project findById(UUID id);
     void save(Project project);
+
```

Comment:
delete this empty line

Reply:
I've deleted the line. I believe this is a styling error and would be great to have a Gradle tasks that handles this. Spotless (com.diffplug.spotless) would be a great option
<!-- Reviewable comment -OD37G643P91DJV2V3hJ:b-bcnvv1 -->
<!-- Sent from Reviewable.io -->